### DISCUSSION TITLE
Limited solution length

### DISCUSSION THREAD

**Discussion by DeveloperAnonymous** [community/community#8184] [Copilot Conversations]
I've come to notice that solutions have a maximum character length, maybe it should be something configurable so we can increase or decrease it if we want.. 

I've come to a situation where there were too many things to generate that the solution was cut and we couldn't see/accept the full solution.

I'm aware that these situations are generally not very commun, but they can happen and it would be amazing to be able to configure it.

thanks

**Comment by tboby** [comment]
I agree, I recently ran into this when using copilot to generate static properties for every value of an enum in C#:
![image](https://user-images.githubusercontent.com/447391/144596981-2f98d0f3-9630-4127-bfeb-1b7af56dc1dc.png)
If this supported longer solutions it would be incredible!
And there doesn't seem to be a way to make it trigger in block mode not line mode once you accept the first part so you can't workaround it that way.

**Reply by andraz** [reply]
Same problem happens in JS if you try to create an array of countries.

**Reply by ajax484** [reply]
> Same problem happens in JS if you try to create an array of countries.

Exactly
I've noticed that it generates 15 solutions, some of which are similar to each other.
I think it would be better if it generated less solutions and the character limit was increased
